en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/choreo-connect-overview.md

```diff
@@ -8,7 +8,7 @@ The WSO2 Choreo Connect is a lightweight message processor for APIs. The Choreo
 
 ## Choreo Connect Components
 
-Choreo Connect has three basic components. Each component does a vital part during request processing, and single unit of Choreo Connect is a collection of all three components.
+Choreo Connect comprises of the following three components, each of which play a vital role when processing requests.
```

Comment:
```suggestion
Choreo Connect consists of the following three components, each of which plays a vital role when processing requests.
```

Reply:
I don't think we need the 'each of which plays a vital role when processing requests' part.